examples/models/lstm_model.py

```diff
@@ -18,9 +18,9 @@ def __init__(self, vocab_size: int, vocab_dimension: int = 128, lstm_dimension:
             bidirectional=True,
         )
         self.drop = nn.Dropout(p=0.3)
-        self.fc = nn.Linear(2 * lstm_dimension, 41)
+        self.fc = nn.Linear(2 * lstm_dimension, 4)
```

Comment:
Moving from the previous new classification task, which had 41 labels, to the AG news task, which only has 4